dist/js/selectmenu.js

```diff
@@ -408,10 +410,7 @@
 			var list = this.list,
 				selected = list.find('[aria-selected="true"]');
 
-			this.trigger.attr({
-				'aria-controls': list.attr('id') || '',
-				'aria-activedescendant': selected.attr('id') || ''
-			});
+			this.trigger.find('input').attr('aria-controls', list.attr('id') || '');
```

Comment:
Will you change this to the below? We already have a handle on the input element.

this.input.attr('aria-controls', list.attr('id') || '');